woocommerce/woocommerce-android
# Product release 1/update pricing

Fixes #1774. This PR adds the option to update the product pricing info.

**Note that this feature is only enabled in debug builds, so end users won't see it yet.**

~- Adding the `Not ready for merge` label since this [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1495) needs to be merged before this PR can be merged.~

#### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/74737093-2183e080-527a-11ea-8695-5a2c9a01adf8.gif">

#### Testing
- Update the product pricing details and click on Update and verify that the Product pricing is updated. - Schedule a sale or cancel a scheduled sale and verify that it is updated correctly.
- Verify that discard is working as expected.

Update release notes:
